packages/inventory-compliance/src/PresentationalComponents/UnsupportedSSGVersion.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Popover, Text } from '@patternfly/react-core';
+import { ExclamationTriangleIcon, OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const UnsupportedSSGVersion = ({ supported = true, ssgVersion }) => {
+    const bodyContent = 'This system is running an unsupported version of the SCAP Security Guide (SSG).' +
+        'This information is based on the last report uploaded for this system to the Compliance service.';
+    const supportedConfigsLink = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/' +
+        'html/assessing_and_monitoring_security_policy_compliance_of_rhel_systems/' +
+        'compl-assess-overview-con#compl-assess-supported-configurations-con';
+    const footerContent = <a target='_blank' rel='noopener noreferrer' href={ supportedConfigsLink }>Supported SSG versions</a>;
+    const divStyle = {
+        display: 'block',
+        margin: '1rem -1.5rem -1.5rem -1.5rem'
+    };
+
+    return !supported && <div className='pf-c-alert pf-m-warning pf-m-inline' style={ divStyle }>
+        <ExclamationTriangleIcon className='ins-u-warning' style={ { marginRight: '.25em' } } />
```

Comment:
> Observation: Fixed width and custom margins. Fixed values are not good for responsiveness.

All units are either em or rem, both variable units. Margins, paddings, width, height etc. basically all properties concerned box sizing should use rem, to be based off the base font size. Here the instance is a bit different and it should inherit the current parents font size to emulate a thin space based on the current elements font size and not need to use a non breaking space, which would be undesirable because it could break responsiveness.

Reply:
For the responsiveness, I had in mind the case when the elements cannot fit into their bounding box and start to be pushed to a new line. The side margins would stay in that case. I know that this is not the case here. It is a good practice to always have that in mind.

Picture this:
```
<triangle icon>RIGHT_MARGIN_WHITESPACE Text... 
LEFT_MARGIN_WHITESPACE<question mark icon>
```
In this picture, the question mark would be pushed to a new line whilst keeping the left margin. The whitespace for that icon would be undesirable.

Reply:
Using a margin this way is nothing bad. In the case you are describing it would be in order to change/remove it using appropriate media-queries to adjust.